docs/core/app-types.md

```diff
@@ -15,126 +15,5 @@ ms.assetid: 93488a0b-c94c-4ed6-97ea-571bb23a862e
 .NET Core App Types
 ===================
 
-## Introduction
-There are several ways to think about the "types" of applications you can build. Usually, the types describe 
-a certain execution model or are based on what the application can "do"; examples of these are "console application", 
-"web application", etc. All of these types of applications (and more) can be created with .NET Core, since it is 
-a general purpose development platform.
-
-However, given its unique and cross-platform nature, .NET Core also has another angle through which to observe the type of the application 
-and that is the application's *portability*. Portability essentially means where you can run your application and what 
```

Comment:
better: ...to observe the type of the application: the application's *portability*.